stdlib/public/SDK/IOKit/IOKit.swift

```diff
@@ -0,0 +1,125 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2016 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+@_exported import IOKit
+
+/// General error
+public let kIOReturnError           = iokit_common_err(0x2bc)
```

Comment:
Would you mind changing them to computed vars?  This would decrease the number of exported symbols.